go.mod

```diff
@@ -59,7 +59,7 @@ require (
 	github.com/aws/aws-sdk-go-v2/internal/configsources v1.1.10 // indirect
 	github.com/aws/aws-sdk-go-v2/internal/endpoints/v2 v2.4.4 // indirect
 	github.com/aws/aws-sdk-go-v2/internal/ini v1.3.11 // indirect
-	github.com/aws/aws-sdk-go-v2/service/ecr v1.17.4 // indirect
+	github.com/aws/aws-sdk-go-v2/service/ecr v1.17.4 // indirect; dari	o indirect
```

Comment:
Is this comment intentional?

Reply:
No, weird. I'm guessing vim got away from me at some point. Fixed.